src/https_client.c

```diff
@@ -63,63 +79,59 @@ static void https_fetch_ctx_init(https_client_t *client,
                                  const char* data, size_t datalen,
                                  struct curl_slist *resolv,
                                  https_response_cb cb, void *cb_data) {
-  ctx->curl = curl_easy_init();
+  ctx->curl = curl_easy_init(); // if failes, first setopt will fail
```

Comment:
Typo. 
s/failes/fails/

Reply:
Sorry.
I will fix it in next - maybe latest - round :)